pkg/api/types.go

```diff
@@ -209,6 +209,23 @@ type IPVSMode struct {
 	MinSyncPeriod string `yaml:"minSyncPeriod"`
 }
 
+type CoreDNSLocalLimits struct {
```

Comment:
Could we use `ComputeResources` instead?

Reply:
Yep -- great idea!